Libraries/Hotcakes.PaypalWebServices/Hotcakes.PaypalWebServices.csproj

```diff
@@ -75,11 +80,22 @@
       <HintPath>..\..\References\Misc\paypal_base.dll</HintPath>
     </Reference>
     <Reference Include="System" />
+    <Reference Include="System.Configuration" />
     <Reference Include="System.ComponentModel.Composition" />
     <Reference Include="System.Core">
       <RequiredTargetFramework>3.5</RequiredTargetFramework>
     </Reference>
     <Reference Include="System.Data" />
+    <Reference Include="System.Runtime.CompilerServices.Unsafe, Version=4.0.4.1, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\System.Runtime.CompilerServices.Unsafe.4.5.2\lib\netstandard2.0\System.Runtime.CompilerServices.Unsafe.dll</HintPath>
+    </Reference>
+    <Reference Include="System.Runtime.InteropServices.RuntimeInformation, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\System.Runtime.InteropServices.RuntimeInformation.4.3.0\lib\net45\System.Runtime.InteropServices.RuntimeInformation.dll</HintPath>
+      <Private>True</Private>
+      <Private>True</Private>
```

Comment:
There are two instances of this.  

Also, the value for `Private` should only be `True` when the DLL should be copied and distributed with the module on the DNN website when it's installed.  Is that the case for this DLL?  Does it need to be installed for the module to work?

Reply:
In the last code review, I mentioned that the stripe solution was hosted by the Hotcakes project. In the current version, the Stripe.dll file comes from the references folder. I assume they compiled Stripe's solution to generate this solution with all its dependencies in a single dll file. With the via NuGet approach, all dependencies must be added. This must be the reason why the new references appear. I checked these packages on the nuget.org site and they are stripe.net dependencies. These dlls are generated as individual files, so they should be copied for stripe to work correctly.

Reply:
Maybe we should go over this together during a meeting.  I think you're probably doing the correct thing, but it's also important that you know the differences and why.